src/core/Akka.Remote/MessageSerializer.cs

```diff
@@ -41,16 +42,16 @@ public static object Deserialize(ActorSystem system, SerializedMessage messagePr
         /// <returns>SerializedMessage.</returns>
         public static SerializedMessage Serialize(ActorSystem system, Address address, object message)
         {
-            Serializer serializer = system.Serialization.FindSerializerFor(message);
```

Comment:
The `FindSerializerFor` has null check, but `FindSerializerForType` don't. I don't know how it will behave with nulls